metau-capstone/Delegates/SceneDelegate.m

```diff
@@ -22,14 +25,30 @@ - (void)scene:(UIScene *)scene willConnectToSession:(UISceneSession *)session op
     PFUser *user = [PFUser currentUser];
     if (user != nil) {
         NSLog(@"Welcome back %@ 😀", user.username);
-
-        // Load Chat view controller and set as root view controller
```

Comment:
Curious, why did you remove this comment?

Reply:
I think I copied some code from Parse chat project and carried that comment over. I can add a comment on loading the study view controller!